server/trillian_log_server/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang
+
+ENV DB_USER test
+ENV DB_PASSWORD zaphod
+ENV DB_DATABASE test
+ENV DB_HOST 127.0.0.0:3306
+
+ENV HOST 0.0.0.0
+ENV RPC_PORT 8090
+ENV HTTP_PORT 8091
+
+ENV DUMP_METRICS 0s
```

Comment:
[It'd be better for caching to set all of these using a single `ENV` command.](https://docs.docker.com/engine/reference/builder/#env)